docs/content/en/highlights/2021-07-21-0-16-upgrade-guide.md

```diff
@@ -44,3 +48,88 @@ inputs = ["datadog_agent"]
 ```
 
 Encoding fields other than `codec` are still valid.
+
+### Vector source/sink version 2 released {#vector-source-sink}
+
+We've released a new major version (`v2`) of our `vector` [source][]/[sink][]
+components. This release resolves a lot of issues we saw with our previous
+(`v1`) TCP-based implementation of these two components.
+
+The new version transitions to using gRPC over HTTP as its communication
+protocol, which resolves those issues.
```

Comment:
Can we just list the issues here? Otherwise people aren't sure what the benefits are and why they should upgrade.